prime-router/docs/design/design/data-model.md

```diff
@@ -0,0 +1,355 @@
+- report_index - the index of the item in the linked report_file the metadata belongs to
+- message_id - unique message ID set by the sender in the item itself
+- previous_message_id - pointer/link to the unique id of a previously submitted item.  Usually blank. Or, if an item modifies/corrects a prior item, this field holds the message_id of the prior item.
+
+The following fields are the actual values contained within an item that a receiver may find useful and each field's purpose can be deduced by its name.
+
+- equipment_model
+- test_result
+- test_result_code
+- ordering_facility_name
+- ordering_facility_city
+- ordering_facility_county
+- ordering_facility_state
+- ordering_facility_postal_code
+- ordering_provider_name
+- ordering_provider_id
+- ordering_provider_county
+- ordering_provider_state
+- ordering_provider_postal_code
+- patient_state
+- patient_county
+- patient_postal_code
+- patient_age
+- patient_gender
+- patient_gender_code
+- patient_ethnicity
+- patient_ethnicity_code
+- patient_race
+- patient_race_code
+- patient_tribal_citizenship
+- patient_tribal_citizenship_code
+- specimen_collection_date_time
+- testing_lab_city
+- testing_lab_county
+- testing_lab_postal_code
+- testing_lab_state
+- testing_lab_name
+- testing_lab_clia
+- created_at
+- site_of_care
+- sender_id
+- test_kit_name_id
+- test_performed_loinc_code
+- organization_name
+- patient_country
+- processing_mode_code
+
+#### `elr_result_metadata` table
+
+There are currently no plans to start using the `elr_result_metadta` table. Metadata tracking in the Universal Pipeline is likely to take an all-together different approach to the metadata question, and may not store metadata in the Postgres database at all, but rather somewhere else where maintainability, extensibility, and searching performance may be improved.
+
+### ReportStream Support Features
+
+This section shall discuss the rest of the database tables that power backend or non-user facing functionality of ReportStream. These tables round out the rest of the features and functionality of ReportStream that the database enables.
+
+#### Associated Tables
+
+- flyway_schema_history
+- jti_cache
+
+#### `flyway_schema_history` table
+
+This table is used to record ReportStream's Flyway migrations and is managed by Flyway. See [Exploring the Flyway Schema History Table](https://www.red-gate.com/hub/product-learning/flyway/exploring-the-flyway-schema-history-table) for more information.
+
+#### `jti_cache` table
+
+The `jti_cache` table is a simple table that keeps track of JWT IDs (JTI) for ReportStream's [server-server authentication implementation](../features/0002-authentication.md#via-smart-on-fhir-oauth-implementation).
+
+The table contains the following columns:
+
+- jti_cache_id
+- jti
+- expires_at
+- created_at
+
+### Deprecated Tables
+
+The following tables deserve a mention since they exist but are not used and thus not worth covering in detail.
+
+- email_schedule
```

Comment:
Would prefer to be given a definition and an explanation why we are keeping it around.

Reply:
Updated to flesh out the documentation here, removed the note about it being deprecated since technically it is not, its just not used currently - and that could change.

Reply:
Reminder on this one.

Reply:
This has been updated since.